docs/howto/openstack/magnum/new-k8s-cluster.md

```diff
@@ -0,0 +1,308 @@
+	for the control plane. Optionally, select one of the available keypairs 
+	for secure SSH access to the individual cluster nodes.
+
+	![Template and keypair](assets/new-k8s/shot-03.png)
+
+	For now, you may skip the _Advanced Option_ section. Click the 
+	green _Create_ button, and Magnum will start creating the new 
+	Kubernetes cluster. Please note that the whole process may take several 
+	minutes to complete.
+=== "OpenStack CLI"
+	A simple, general command for creating a new Kubernetes cluster 
+	with Magnum looks like this:
+	
+	```bash
+	openstack coe cluster create \
+		--cluster-template $CLUSTER_TMPL \
+		--keypair $KEYPAIR \
+		$CLUSTER_NAME
+	```
+
+	Let us list all available templates in the region:
+	
+	```bash
+	openstack coe cluster template list
+	```
+
+	```plain
+	+--------------------------------------+----------------------------------------------------------------+------+
+	| uuid                                 | name                                                           | tags |
+	+--------------------------------------+----------------------------------------------------------------+------+
+	| 3f476f01-b3de-4687-a188-6829ed947db0 | Kubernetes 1.15.5 on Fedora-atomic 29 4C-8GB-20GB No Master LB | None |
+	| c458f02d-54b0-4ef8-abbc-e1c25b61165a | Kubernetes 1.15.5 on Fedora-atomic 29 2C-4GB-20GB No Master LB | None |
+	| f9e1a2ea-b1ff-43e7-8d1e-6dd5861b82cf | Kubernetes 1.18.6 on Fedora-coreos 33 2C-4GB-20GB No Master LB | None |
+	+--------------------------------------+----------------------------------------------------------------+------+
+	```
+
+	Select the template you want by setting the corresponding 
+	`uuid` value to the  `CLUSTER_TMPL` variable:
+	
+	```bash
+	CLUSTER_TMPL="f9e1a2ea-b1ff-43e7-8d1e-6dd5861b82cf" # just an example
+	```
+	
+	Then, list all available keypairs...
+	
+	```bash
+	openstack keypair list
+	```
+
+	```plain
+	+---------+-------------------------------------------------+------+
+	| Name    | Fingerprint                                     | Type |
+	+---------+-------------------------------------------------+------+
+	| husavik | 34:3b:58:ba:ec:95:f5:17:17:df:04:38:11:89:e6:3d | ssh  |
+	+---------+-------------------------------------------------+------+
+	```
+	
+	...and set to the `KEYPAIR` variable the name of the keypair 
+	you want:
+	
+	```bash
+	KEYPAIR="husavik" # again, this is just an example
+	```
+	
+	Finally, decide on a name for your new Kubernetes cluster:
+	
+	```bash
+	CLUSTER_NAME="bangor" # just remembered a famous storyteller
+	```
+	
+	With everything in place, go ahead and create your new 
+	Kubernetes cluster:
+	
+	```bash
+	openstack coe cluster create \
+		--cluster-template $CLUSTER_TMPL \
+		--keypair husavik
+		bangor
+	```
+	If everything went well with your request for a new cluster, 
+	on your terminal, you would see a message like the following:
+	 
+	 ```plain
+	 Request to create cluster e0df8c62-c6f6-4c7d-b67e-33e3606e9ab6 
+	accepted
```

Comment:
the "accepted" line appears to be oddly indented with one less space?

Reply:
Yeah, that's due to the `fold` tool and my manual edits :/